doc/internal/man3/OSSL_METHOD_STORE.pod

```diff
@@ -102,6 +102,12 @@ The I<method_up_ref> function is called to increment the
 reference count of the method and the I<method_destruct> function is called
 to decrement it.
 
+=head1 NOTES
+
+The I<prop_query> argument to ossl_method_store_cache_get() and
+ossl_method_store_cache_set() is not allowed to be NULL.  Use "" for an
```

Comment:
This could almost go in the RETURN values section.. i.e NULL returns 0

Reply:
I think the stronger wording of just **don't do this** is better.
I could do both which would be even better.